contributing/code/bc.rst

```diff
@@ -267,6 +275,13 @@ Turn static into non static                         No
 .. [4] When changing the parent class, the original parent class must remain an
        ancestor of the class.
 
+.. [5] The value of a constant may only be changed when the constants aren't
+       used in configuration (e.g. Yaml and XML files), as these do not support
```

Comment:
Yaml should be removed until https://github.com/symfony/symfony/pull/18626 is merged in 3.2